### DISCUSSION TITLE
Что означает ошибка recv: Connection reset by peer?

### DISCUSSION THREAD

**Discussion by artem-burashnikov** [hufrea/byedpi#53] [Q&A] [answered]
Можете пояснить, пожалуйста, что означает это сообщение в терминале.
Периодически появляется, но видимых проблем не замечаю.

**Comment by hufrea** [comment, marked as answer]
Соединение сброшено. Обычно эта ошибка возникает, когда DPI блокирует соединение.

**Reply by artem-burashnikov** [reply]
Можете пояснить, пожалуйста, почему тогда **видимых** нарушений я не наблюдаю? Устанавливается какое-то другое соединение?

**Reply by hufrea** [reply]
Если используется параметр `--auto` (без none), то устанавливается новое соединение с другими параметрами.

**Reply by artem-burashnikov** [reply]
Использую просто `--oob 1`

**Reply by artem-burashnikov** [reply]
Ну, видимо, много чего происходит за кулисами в принципе, просто хотел узнать немного деталей. Спасибо.

**Reply by hufrea** [reply]
Тогда, возможно, не загружаются малозаметные элементы (аватарки в youtube или еще что-то). Вы можете запустить программу с параметром `--debug 1` и посмотреть какие именно домены/IP не открываются.
